backend/api/src/tests/api/v1/gql/chat.rs

```diff
@@ -0,0 +1,153 @@
+use async_graphql::{Name, Request, Variables};
+use chrono::Utc;
+use common::types::{session, user};
+use std::sync::Arc;
+
+use crate::{
+    api::v1::gql::{request_context::RequestContext, schema},
+    tests::global::mock_global_state,
+};
+
+#[tokio::test]
+async fn test_send_message() {
+    // Set up global state
+    let (global, _handler) = mock_global_state(Default::default()).await;
+
+    sqlx::query!("DELETE FROM chat_messages")
+        .execute(&*global.db)
+        .await
+        .unwrap();
+
+    sqlx::query!("DELETE FROM chat_rooms")
+        .execute(&*global.db)
+        .await
+        .unwrap();
+
+    sqlx::query!("DELETE FROM sessions")
+        .execute(&*global.db)
+        .await
+        .unwrap();
+
+    sqlx::query!("DELETE FROM users")
+        .execute(&*global.db)
+        .await
+        .unwrap();
+
+    // not logged in
+    let schema = schema();
+
+    let query = r#"
+        mutation {
+            chat {
+                sendMessage(chatId: 1, content: "message")
+            }
+        }
+    "#;
+
+    let ctx = Arc::new(RequestContext::default());
+
+    let res = schema
+        .execute(Request::from(query).data(global.clone()).data(ctx))
+        .await;
+
+    assert_eq!(res.errors.len(), 1);
```

Comment:
Can we check the error contents, please refer to other tests for how this is done